### PR TITLE
parse unicode forall and exists as identifiers

### PR DESCRIPTION
**Pull request by willow-ahrens**
A proposed fix to to #19012, allowing `∀` and `∃` to be parsed as identifier characters, just like `∑`, `∫`, `⋃`, or `Π`.

### CODE REVIEW

**Comment by stevengj** [comment]
I'm in favor.  No one has proposed a convincing infix usage for these AFAICT.

File: NEWS.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ New language features
   to enforce the involved function calls to be (or not to be) inlined. ([#41312])
 * The default behavior of observing `@inbounds` declarations is now an option via `auto` in `--check-bounds=yes|no|auto` ([#41551])
 * New function `eachsplit(str)` for iteratively performing `split(str)`.
+* `∀` and `∃` are now allowed as identifier characters.
```
Comment (stevengj) [inline NEWS.md:15]:
```suggestion
* `∀` and `∃` are now allowed as identifier characters ([#42314]).
```

**Comment by JeffBezanson** [comment]
Triage approves, and suggests adding U+2204 "there does not exist" as well.

**Comment by peterahrens** [comment]
I've added the "not exists" (`∄`) character. I was unable to find a "not forall" character, and upon reflection, I'm unsure what it would even look like.

**Comment by peterahrens** [comment]
To make the conclusion of my last comment a little more clear: I believe this PR is ready for another review or a merge.

**Review by stevengj — APPROVED** [review]

**Pull request merged**